README.md

```diff
@@ -116,7 +116,7 @@ Different subsystems have different test suites. Running these may require root
 Example, from `$GOPATH/src/github.com/armPelionEdge/maestro`:
 
 ```
-DEBUG=1 DEBUG2=1 ./build.sh && cd networking && sudo \
+DEBUG=1 DEBUG2=1 ./build.sh && cd networking && sudo -E \
```

Comment:
We need -E here so that the environment($GOROOT, $GOBIN, $GOPATH) is preserved when using sudo as well.